features/users/router.py

```diff
@@ -0,0 +1,107 @@
+"""Users feature endpoints"""
+from typing import List
+
+import fastapi
+from fastapi import APIRouter, HTTPException
+from starlette.responses import JSONResponse
+
+import features.users.exceptions
+from .input_models import RegisterUserInputModel, UpdateUserInputModel
+from .responses import UsersResponseModel
+
+from .operations import create_token, create_new_user, signin_user, get_all_users, get_user
+
+user_router = APIRouter()
+
+
+@user_router.post("/signup")
+async def signup(user: RegisterUserInputModel):
+    """
+    Sing up user
+
+    :param user:
+    :return:
+    """
+    try:
+        db_user = create_new_user(user)
+        return JSONResponse(
+            content={
+                "message": "Successfully created",
+                "user_data": {"id": db_user.id, "username": db_user.username, "email": db_user.email}
```

Comment:
Return just the response model. You don`t need to add redundant information such "Successfully created", returning status code 200 is already saying that.